src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -49,6 +52,19 @@ private struct Element
             public const string GridHeightAnimator = "gridHeightAnimator";
         }
 
+        /// <summary>
+        /// TextBlock, which is shown during graph calculation.
+        /// TODO: this should be replaced with some animated text. 
+        /// http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-9584
+        /// </summary>
+        private readonly TextBlock busyTextBlock = new TextBlock
+        {
+            Text = Configurations.BusyString,
+            FontFamily = new FontFamily("Consolas"),
+            Opacity = 0.5,
+            Margin = new System.Windows.Thickness(12, 6, 0, 10)
+        };
+
```

Comment:
Instead of `TextBlock`, can we not just use a `MirrorData` that contains a `String` that is `...`, and use the regular watch view to show it? That way we don't have to manage another `TextBlock` just for this.